### ISSUE TITLE
Undefined Method `manage_seo?'

### ISSUE THREAD

**Issue by NySiya** [owen2345/camaleon-cms-seo#1]
Hi @owen2345,

After install Camaleon CMS SEO plugin on the theme, and than I select any page in order to manage the Seo Setting for the page and I get this errors: 
```
undefined method `manage_seo?' for #<CamaleonCms::Post:0x007fd5959790d8>
Did you mean?  manage_keywords?
```

Please, check the attachment below

<img width="750" alt="screen shot 2016-12-08 at 10 30 50 am" src="https://cloud.githubusercontent.com/assets/12697539/20996628/70479892-bd31-11e6-9d39-232dd6b3cf93.png">

**Comment by owen2345** [comment]
@NySiya I added support for old versions of Camaleon CMS, please use the git version to confirm me if its works well for you too and release a new version.
Regards,

**Comment by NySiya** [comment]
@owen2345 Thank you so much. After I used Git version it show me this 
<img width="1068" alt="screen shot 2016-12-09 at 9 20 26 am" src="https://cloud.githubusercontent.com/assets/12697539/21035330/cf711c20-bdf0-11e6-8e88-5517c07c2e42.png">

**Comment by NySiya** [comment]
Hello @owen2345, I have fixed the problem by change from post to args[:post] like below:
`unless cama_meta_tag_post_is_for_old_version?(post)
      manage_seo = args[:post].manage_seo?
    else
      manage_seo = args[:post].manage_keywords?(args[:post_type]) # support for old Camaleon CMS versions (use keywords setting instead of seo setting)
    end`
to
`unless cama_meta_tag_post_is_for_old_version?(args[:post])
      manage_seo = args[:post].manage_seo?
    else
      manage_seo = args[:post].manage_keywords?(args[:post_type]) # support for old Camaleon CMS versions (use keywords setting instead of seo setting)
    end`

**Comment by NySiya** [comment]
After I did all of that. It shows me that I could add/edit values SEO Settings but The Big Problem is the Data in SEO Setting is not save at all, as I check in database, which means all of fields in SEO Setting are not render by connected to any database column.

**Comment by owen2345** [comment]
yes, vs 1.2 is working well.
try gem 'cama_meta_tag', '1.2'

**Comment by NySiya** [comment]
Yeah :) Thank for that. It is working fine. But the problem is on Each Post Type Setting, I did not see such any thing as Enable or Disable SEO as you describe on Read Me.

**Comment by NySiya** [comment]
It is working fine, now. Thank you so much :) ... I have also create pull request in the gem. Please, check.
